GitUI/BranchTreePanel/RepoObjectsTree.Nodes.Remotes.cs

```diff
@@ -287,6 +287,11 @@ public bool Prune()
                 return DoPrune();
             }
 
+            public void OpenRemoteUrlInBrowser()
+            {
+                Process.Start(_remote.FetchUrl);
```

Comment:
I think this only makes sense for http, but won't make sense for other types of urls, e.g. ssh or git.

Reply:
@RussKie whats your suggestion? I'd like to hear and will implement/change it

Reply:
First, in the current version, the menu item should only be available for Http remotes.
However, there is a framework for handling ssh remotes too, currently only GitHub I believe. Se ExternalLink, implemented by @pmiossec that may have some suggestions.

Reply:
@pmiossec any suggestions? 😄

Reply:
Maybe you could use `GitProtocolExtensions.IsUrlUsingHttp()` and indeed disable it if that's not `true`.

Reply:
@pmiossec by disabling you mean to hide/deactivate the button?  if not, please specify. Thanks!

Reply:
Yes. Hide or deactivate. It not always easy to choose between the 2 and in your case, I'm not sure to know which one make more sense.

I tend to prefer disabling the menuitem (so `Enabled = false`) because that give a kind of feedback to the user.

But I would like the others to give there point of view... (@gerhardol was just asking for my advice on the way to determine the kind of url, not especially on my UI preferences 😉 )

Reply:
@pmiossec I was more referring to your other external link changes actually
In this case I prefer hide, most users will just see a disabled item assuming ssh is more common still (http credential store error messages are not useful).

Reply:
Yes, only show the menu for HTTP remotes. 
Please use `GitProtocolExtensions.IsUrlUsingHttp()` to detect those.